src/python/grpcio/grpc/_cython/_cygrpc/credentials.pxd.pxi

```diff
@@ -97,3 +97,8 @@ cdef class ServerCredentials:
   cdef object cert_config_fetcher
   # whether C-core has asked for the initial_cert_config
   cdef bint initial_cert_config_fetched
+
+
+cdef class LocalChannelCredentials(ChannelCredentials):
+
+  cdef readonly object _local_connect_type
```

Comment:
- Why `object` and not `grpc_local_connect_type`?
- `readonly` is not like `const` in Cython. `readonly` creates an additional property getter on the type that returns that member. It should be unnecessary here and just takes more memory.

Reply:
Good suggestions.